### DISCUSSION TITLE
Copying `Data` to `void*`

### DISCUSSION THREAD

**Discussion by DUOLabs333** [capnproto/capnproto#1917] [Q&A] [answered]
If it was `Text`, I could just do a memcpy from `cStr()` --- however, I'm dealing with arbitary bytes, so I can't guarentee it will end with a null-terminator. Could I just do `.asBytes().begin()`?

**Comment by kentonv** [comment, marked as answer]
You don't need the `.asBytes()`, since `Data` is already bytes. Just `.begin()` will give you a pointer to the beginning of the bytes.
